bundles/org.openhab.ui/web/src/components/item/metadata/item-metadata-alexa.vue

```diff
@@ -1,35 +1,65 @@
-        <select name="parameters" @change="updateClasses" :multiple="multiple">
+    <f7-list v-if="ready">
+      <f7-list-item
+        :key="classSelectKey"
+        :title="'Alexa Device Type' + (itemType !== 'Group' ? !multiple ? '/Attribute' : '/Attributes' : '')"
+        smart-select
+        :smart-select-params="{ openIn: 'popup', searchbar: true, closeOnSelect: !multiple, scrollToSelectedItem: true }"
+        ref="classes">
+        <select v-if="itemType === 'Group'" name="classes" @change="updateClasses">
+          <option value="" />
+          <option v-for="cl in orderedClasses" :value="cl" :key="cl" :selected="isSelected(cl)">
+            {{ cl }}
+          </option>
+        </select>
+        <select v-else name="classes" @change="updateClasses" :multiple="multiple">
           <option v-if="!multiple" value="" />
-          <optgroup label="Labels" v-if="!multiple && itemType !== 'Group'">
-            <option v-for="cl in orderedClasses.filter((c) => c.indexOf('label:') === 0)"
-                    :value="cl.replace('label:', '')" :key="cl"
-                    :selected="isSelected(cl.replace('label:', ''))">
-              {{ cl.replace('label:', '') }}
+          <optgroup label="Default Attributes" v-if="!multiple">
+            <option v-for="cl in defaultClasses" :value="cl" :key="cl" :selected="isSelected(cl)">
+              {{ cl }}
+            </option>
+          </optgroup>
+          <optgroup label="Specific Attributes">
+            <option v-for="cl in specificClasses" :value="cl" :key="cl" :selected="isSelected(cl)" :disabled="isDefined(cl)">
+              {{ cl }}
             </option>
           </optgroup>
-          <optgroup label="Capabilities">
-            <option v-for="cl in orderedClasses.filter((c) => c.indexOf('label:') !== 0 && c.indexOf('endpoint:') === (itemType === 'Group'? 0 : -1))"
-                    :value="cl.replace('endpoint:', '')" :key="cl"
-                    :selected="isSelected(cl.replace('endpoint:', ''))">
-              {{ cl.replace('endpoint:', '') }}
+          <optgroup label="Generic Attributes" v-if="!multiple">
+            <option v-for="cl in genericClasses" :value="cl" :key="cl" :selected="isSelected(cl)">
+              {{ cl }}
             </option>
           </optgroup>
         </select>
       </f7-list-item>
+      <f7-block-footer class="padding-left no-padding no-margin" v-if="item.groups.length">
+        <small>Part of group endpoint{{ item.groups.length > 1 ? 's' : '' }}: {{ groupsDescription }}</small>
+      </f7-block-footer>
     </f7-list>
-    <div>
+    <div v-if="ready">
       <config-sheet :parameterGroups="[]" :parameters="parameters" :configuration="metadata.config" />
     </div>
+    <f7-block class="padding-top no-padding no-margin" v-if="itemType === 'Group' && classes.length">
```

Comment:
Please add a `margin-left` class to make it look better when the screen is narrow:

![image](https://user-images.githubusercontent.com/2004147/132695766-40b22725-50fd-4434-be04-883bbec81ea0.png)

Reply:
Done. I added `padding-left` class to the `f7-block-title` and `f7-block-footer` tags part of that block.